### PR TITLE
Write sc-machine idea in readme

### PR DESCRIPTION
**Pull request by NikitaZotov**
* [x] Read PR [documentation](https://github.com/ostis-ai/sc-machine/blob/main/docs/dev/pr.md/)
* [x] Update changelog
* [x] Update documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,23 +4,57 @@
 [![codecov](https://codecov.io/gh/ostis-ai/sc-machine/branch/main/graph/badge.svg?token=WU8O9Z1DNL)](https://codecov.io/gh/ostis-ai/sc-machine)
 [![docker](https://img.shields.io/docker/v/ostis/sc-machine?arch=amd64&label=Docker&logo=Docker&sort=date)](https://hub.docker.com/r/ostis/sc-machine)
 
-# Documentation: [GitHub Pages](https://ostis-ai.github.io/sc-machine)
+<b>SC-machine</b> is a software implementation of semantic network storage and processing of information in language
+of universal knowledge representation SC-code, provided by [<b>the OSTIS Technology</b>](https://github.com/ostis-ai/ostis-standard). 
+SC-machine is a <u>platform-independent graph database management system</u> to store, retrieves and run tasks (agents) 
+on knowledge graphs. All declarative (data, data structures, documentation, tasks specification, etc.) and procedural 
+(programs, modules, systems, communication between systems) knowledge represents on one language, named as SC-code.
```
Comment (MksmOrlov) [inline README.md:11]:
```suggestion
(programs, modules, systems, communication between systems) knowledge represents on the same language, named as SC-code. 
```

File: README.md [hunk 2]
```diff
@@ -4,23 +4,57 @@
...
 [![docker](https://img.shields.io/docker/v/ostis/sc-machine?arch=amd64&label=Docker&logo=Docker&sort=date)](https://hub.docker.com/r/ostis/sc-machine)
 
-# Documentation: [GitHub Pages](https://ostis-ai.github.io/sc-machine)
+<b>SC-machine</b> is a software implementation of semantic network storage and processing of information in language
+of universal knowledge representation SC-code, provided by [<b>the OSTIS Technology</b>](https://github.com/ostis-ai/ostis-standard). 
+SC-machine is a <u>platform-independent graph database management system</u> to store, retrieves and run tasks (agents) 
+on knowledge graphs. All declarative (data, data structures, documentation, tasks specification, etc.) and procedural 
+(programs, modules, systems, communication between systems) knowledge represents on one language, named as SC-code. 
+Nowadays sc-machine includes:
+- semantic storage to store graph constructions, named as sc-memory (graph database);
```
Comment (MksmOrlov) [inline README.md:13]:
```suggestion
- semantic storage to store graph constructions, named sc-memory (graph database);
```

File: README.md [hunk 3]
```diff
@@ -4,23 +4,57 @@
...
+- event-based agents processing, named as sc-event-manager (asynced programs workflows handling);
+- tools to communicate with storage:
+   - <u>native</u> program API on C++;
+   - <u>network</u> program API, provided by <i>sc-server</i> to communicate with storage by network with help of Websocket
```
Comment (MksmOrlov) [inline README.md:17]: I think "program" is redundant

File: README.md [hunk 4]
```diff
@@ -4,23 +4,57 @@
...
+   - <u>network</u> program API, provided by <i>sc-server</i> to communicate with storage by network with help of Websocket 
+     and JSON languages;
+   - <i>sc-builder</i> to load information from SCs files into storage (SCs-code is a one of external representation 
+     languages of SC-code);
+
+SC-machine provide complete and sufficient native and network APIs to create, update, delete and retrieve operations and
+also supports event-driven communication with sc-memory and sc-agents, located in it.
+
+Start use [OSTIS-web-platform](https://github.com/ostis-ai/ostis-standard) to integrate all the world solutions <u>by the 
+way of one technology</u>, <u>one data representation and programming language</u> and <u>one program software</u>!
```
Comment (MksmOrlov) [inline README.md:26]: It seems that u tag doesn't work. Chack all over the document

File: README.md [hunk 5]
```diff
@@ -4,23 +4,57 @@
...
+# Documentation: 
+
+- A part of sc-machine documentation, represented in md format: [GitHub Pages](https://ostis-ai.github.io/sc-machine).
+  It serves as a guide for installation, development and a brief description of the project modules.
+- SCn-code represented sc-machine documentation: [LaTeX documentation](https://github.com/ostis-ai/ostis-web-platform/blob/develop/docs/main.pdf).
+  This documentation fully describes the sc-machine: its components, software interfaces for interacting with sc-memory, 
+  the characteristics and design of sc-memory, as well as the rationale for the sc-machine tools and solutions used.
 
 # Quick start
+Sc-machine is a part of the whole system, named as <b>OSTIS Web platform</b>.
```
Comment (MksmOrlov) [inline README.md:37]: What do you mean by "whole system"? I think it is better to say that sc-machine is a core for an ostis-system. OSTIS Web Platform is an example of one

File: README.md [hunk 6]
```diff
@@ -4,23 +4,57 @@
...
 You can use our [OSTIS Web platform](https://github.com/ostis-ai/ostis-web-platform) project to get up and running quickly.
 # Installation
 ## Docker
-We're providing a Docker image for this project. Head to [Installing with Docker](https://ostis-ai.github.io/sc-machine/docker) section of our docs to try it out!
+We're providing a Docker image for this project. Head to [Installing with Docker](https://ostis-ai.github.io/sc-machine/docker) 
+section of our docs to try it out! We recommend to use this way for sc-machine deployment. <u>When error or problem occurred</u>,
+[write issue](https://github.com/ostis-ai/sc-machine/issues) to OSTIS-web-platform developers.
```
Comment (MksmOrlov) [inline README.md:43]: Why this line is in Docker section?

File: README.md [hunk 7]
```diff
@@ -4,26 +4,26 @@
...
+   - *sc-builder* to load information from SCs files into storage (SCs-code is a one of external representation 
      languages of SC-code);
 
 SC-machine provide complete and sufficient native and network APIs to create, update, delete and retrieve operations and
 also supports event-driven communication with sc-memory and sc-agents, located in it.
 
-Start use [OSTIS-web-platform](https://github.com/ostis-ai/ostis-standard) to integrate all the world solutions <u>by the 
-way of one technology</u>, <u>one data representation and programming language</u> and <u>one program software</u>!
+Start use [OSTIS-web-platform](https://github.com/ostis-ai/ostis-standard) to integrate all the world solutions ***by the 
+way of one technology***, ***one data representation and programming language*** and ***one program software***!
```
Comment (MksmOrlov) [inline README.md:26]: I am not sure that "one" is correct word here. Maybe "the same" or smth like that

**Review by MksmOrlov — APPROVED** [review]

File: README.md [hunk 8]
```diff
@@ -4,31 +4,70 @@
...
+- semantic storage to store graph constructions, named sc-memory (graph database);
+- event-based agents processing, named sc-event-manager (asynced program workflows handling);
+- tools to communicate with storage:
+   - *native* API on C++;
+   - *network* API, provided by *sc-server* to communicate with storage by network with help of Websocket 
+     and JSON languages;
+   - *sc-builder* to load information from SCs files into storage (SCs-code is a one of external representation 
+     languages of SC-code);
+
+SC-machine provide complete and sufficient native and network APIs to create, update, delete and retrieve operations and
```
Comment (AlexandrZagorskiy) [inline README.md:22]:
```suggestion
SC-machine provide complete and sufficient native and network APIs to create, retrieve, update and delete  operations and
```

File: README.md [hunk 9]
```diff
@@ -4,31 +4,70 @@
...
+also supports event-driven communication with sc-memory and sc-agents, located in it.
+
+Start use [OSTIS-web-platform](https://github.com/ostis-ai/ostis-standard) to integrate all the world solutions ***by the
```
Comment (AlexandrZagorskiy) [inline README.md:25]:
```suggestion
Start use [OSTIS-web-platform](https://github.com/ostis-ai/ostis-web-platform) to integrate all the world solutions ***by the 
```

**Review by AlexandrZagorskiy — CHANGES_REQUESTED** [review]

File: README.md (hunk 8 above)
Comment (MksmOrlov) [reply README.md:22]: why?

**Review by AlexandrZagorskiy — APPROVED** [review]

**Pull request merged**